docs/package.json

```
@@ -40,13 +44,13 @@
     "svelte-preprocess": "^4.10.7",
     "tslib": "^2.3.1",
     "typescript": "^4.7.2",
-    "@sveltejs/adapter-static": "^1.0.0-next.39",
-    "@sveltejs/adapter-vercel": "next",
     "unplugin-icons": "^0.13.4"
   },
   "type": "module",
   "dependencies": {
+    "@uniswap/widgets": "^2.7.0",
     "@web3-onboard/core": "^2.7.0",
-    "@web3-onboard/injected-wallets": "^2.0.16"
+    "@web3-onboard/injected-wallets": "^2.0.16",
```

```suggestion
    "@web3-onboard/injected-wallets": "^2.2.3",
```
For latest wallet support and stability